understanding/20/focus-visible.html

```diff
@@ -28,7 +28,7 @@ <h2>Intent of Focus Visible</h2>
       
       <p>“Mode of operation” accounts for user agents which may not always show a focus indicator, or only show the focus indicator when the keyboard is used. User agents may optimise when the focus indicator is shown, such as only showing it when a keyboard is used. Authors are responsible for providing at least one mode of operation where the focus is visible. In most cases there is only one mode of operation so this success criterion applies. The focus indicator must not be time limited, when the keyboard focus is shown it must remain.</p>
       
-   	<p>Note that a keyboard focus indicator can take different forms.<span class="wcag22"> This criterion does not specify what the form is, but <a href="focus-appearance-minimum">Focus Appearance</a> does define how visible the indicator should be. Passing <a href="focus-appearance-minimum">Focus Appearance (Minimum)</a> would pass this success criterion.</span></p>
+   	<p>Note that a keyboard focus indicator can take different forms.<span class="wcag22"> This criterion does not specify what the form is, but <a href="focus-appearance">2.4.13 Focus Appearance (Level AAA)</a> does define how visible the indicator should be. Passing <a href="focus-appearance">Focus Appearance</a> would pass this success criterion.</span></p>
```

Comment:
I find the existing wording a little awkward. How about...
```suggestion
   	<p>Note that a keyboard focus indicator can take different forms.<span class="wcag22"> While Focus Visible does not specify what the form is, <a href="focus-appearance">2.4.13 Focus Appearance (Level AAA)</a> provides guidance on creating a consistent, visible indicator. Passing <a href="focus-appearance">Focus Appearance</a> will also pass this Success Criterion.</span></p>
```